tests/foreman/ui/test_registration.py

```diff
@@ -121,3 +128,496 @@ def test_negative_global_registration_without_ak(
+        4. open the global registration form and update the gpg repo and key
+        5. check host is registered successfully with installed same package
+        6. check gpg repo is exist in registered host
+
+    :parametrized: yes
+    """
+    client = rhel8_contenthost
+    repo_name = 'foreman_register'
+    repo_url = settings.repos.gr_yum_repo.url
+    repo_gpg_url = settings.repos.gr_yum_repo.gpg_url
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.capsule': default_smart_proxy.name,
+                'general.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+                'advanced.force': True,
+                'advanced.install_packages': 'mlocate vim',
+                'advanced.repository': repo_url,
+                'advanced.repository_gpg_key_url': repo_gpg_url,
+            }
+        )
+
+    # rhel repo required for insights client installation,
+    # syncing it to the satellite would take too long
+    rhelver = client.os_version.major
+    if rhelver > 7:
+        repos = {f'rhel{rhelver}_os': settings.repos[f'rhel{rhelver}_os']['baseos']}
+    else:
+        repos = {
+            'rhel7_os': settings.repos['rhel7_os'],
+            'rhel7_extras': settings.repos['rhel7_extras'],
+        }
+    client.create_custom_repos(**repos)
+    # run curl
+    result = client.execute(cmd)
+    assert result.status == 0
+    result = client.execute('yum list installed | grep mlocate')
+    assert result.status == 0
+    assert 'mlocate' in result.stdout
+    result = client.execute(f'yum -v repolist {repo_name}')
+    assert result.status == 0
+    assert repo_url in result.stdout
+
+
+@pytest.mark.tier3
+@pytest.mark.usefixtures('enable_capsule_for_registration')
+def test_global_re_registration_host_with_force_ignore_error_options(
+    session, module_activation_key, default_os, default_smart_proxy, rhel7_contenthost
+):
+    """If the ignore_error and force checkbox is checked then registered host can
+    get re-registered without any error.
+
+    :id: 8f0ecc13-5d18-4adb-acf5-3f3276dccbb7
+
+    :expectedresults: Verify the force and ignore checkbox options
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. create and sync repository
+        2. create the content view and activation-key
+        3. open the global registration form and select --force and --Ignore Errors option
+        4. registered the host with generated curl command
+        5. re-register the same host again and check it is getting registered
+
+    :parametrized: yes
+    """
+    client = rhel7_contenthost
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.capsule': default_smart_proxy.name,
```

Comment:
```suggestion
```

Reply:
I am going to address all the updates in test in https://github.com/SatelliteQE/robottelo/pull/14001